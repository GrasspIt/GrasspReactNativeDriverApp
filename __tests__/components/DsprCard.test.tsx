//import React from 'react';
//import { render } from '@testing-library/react-native';
//
//import DsprCard from '../../components/DsprCard';
//
//const dspr = {
//  id: 12,
//  name: 'name',
//  deliveryServiceProvider: 12,
//};
//
//const handleSelect = jest.fn();
//
//describe('<DsprCard />', () => {
//  it('renders correctly', async () => {
//    const component = render(<DsprCard dspr={dspr} handleSelect={handleSelect} />).toJSON();
//    await expect(component).toMatchSnapshot();
//  });
//});
