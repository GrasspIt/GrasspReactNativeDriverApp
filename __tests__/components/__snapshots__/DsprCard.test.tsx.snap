// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DsprCard /> renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "elevation": 5,
      "opacity": 1,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.4,
      "shadowRadius": 2,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "borderRadius": 5,
          "display": "flex",
          "elevation": 5,
          "justifyContent": "center",
          "margin": 10,
          "maxWidth": 500,
          "overflow": "hidden",
        },
        Object {
          "backgroundColor": "#ffffff",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "minHeight": 200,
          "padding": 10,
          "width": "100%",
        }
      }
    >
      <Image
        resizeMode="contain"
        source={1}
        style={
          Object {
            "flex": 1,
            "height": undefined,
            "width": undefined,
          }
        }
      />
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "bottom": 0,
            "flex": 1,
            "justifyContent": "center",
            "opacity": 0.9,
            "position": "absolute",
            "width": "100%",
          },
          Object {
            "backgroundColor": "#000000",
          },
        ]
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 20,
            "paddingVertical": 20,
            "textAlign": "center",
          }
        }
      >
        name
      </Text>
    </View>
  </View>
</View>
`;
